datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -242,17 +251,10 @@ CREATE TABLE dcsa_im_v3_0.party_function (
     party_function_description varchar(250) NOT NULL
 );
 
-DROP TABLE IF EXISTS dcsa_im_v3_0.party_contact_details CASCADE;
-CREATE TABLE dcsa_im_v3_0.party_contact_details (
-    id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
-    name varchar(100) NULL,
-    phone varchar(30) NULL,
-    email varchar(100) NULL
-);
-
 DROP TABLE IF EXISTS dcsa_im_v3_0.document_party CASCADE;
 CREATE TABLE dcsa_im_v3_0.document_party (
-    party_id varchar(100) NOT NULL DEFAULT uuid_generate_v4()::text REFERENCES dcsa_im_v3_0.party (id),
+    id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
```

Comment:
I am surprised by the `id` on the document party.  As I understood it, we deliberately did _not_ have a PK on document party.

Reply:
This was actually suggested by @raisoman , maybe he can provide some clarity on it. 
From what I understand though,  we use the `id` in `document_party` as a reference in `displayed_address` since multiple `displayed_address` can be linked to one `document_party` and hence we need the PK in `document_party`.
Note that `id` is not part `document_party` in the current IM document.

Reply:
Reverted the changes for document party and displayed address, we will rework it in IM 3.3.